<launch>
  <machine name="robot1" address="192.168.208.80" env-loader="/opt/ros/noetic/env.sh" user="revast-user" password="temppwd"/>
  <machine name="robot2" address="192.168.208.149" env-loader="/home/revast-user/workspace/kinetic/devel/env.sh" user="revast-user" password="temppwd"/>
  <machine name="robot3" address="192.168.208.193" env-loader="/home/revast-user/workspace/kinetic/devel/env.sh" user="revast-user" password="temppwd"/>
<node name="joy" pkg="joy" type="joy_node" output="screen"/> 
<!-- <node name="jref" pkg="scamper_ota" type="joycon145.py" output="screen"/> 
  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: 192.168.208.10
      port: 3883

      update_frequency: 240.0
      frame_id: map

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: false
      broadcast_tf: true

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 10.0
    </rosparam>
  </node> -->
  
  <group ns="scamper201">
  
    <node machine="scamper201" name="scamper_driver" pkg="ros_scamper_core" type="ros_scamper_driver" respawn="true" output="screen">
      <param name="config_path" value="/home/revast-user/workspace/kinetic/src/ros_scamper_core/config/scamper_3wd.ini" type="string" />
    </node>

  </group>
  <group ns="scamper204">
  
    <node machine="scamper204" name="scamper_driver" pkg="ros_scamper_core" type="ros_scamper_driver" respawn="true" output="screen">
      <param name="config_path" value="/home/revast-user/workspace/kinetic/src/ros_scamper_core/ros_scamper_core/config/scamper_3wd.ini" type="string" />
    </node>

  </group>

  <group ns="scamper205">
  
    <node machine="scamper205" name="scamper_driver" pkg="ros_scamper_core" type="ros_scamper_driver" respawn="true" output="screen">
      <param name="config_path" value="/home/revast-user/workspace/kinetic/src/ros_scamper_core/ros_scamper_core/config/scamper_3wd.ini" type="string" />
    </node>

  </group>


</launch>

